gprofiler/profilers/java.py

```diff
@@ -443,6 +527,29 @@ def _profile_ap_process(self, ap_proc: AsyncProfiledProcess) -> Optional[StackTo
             logger.info(f"Finished profiling process {ap_proc.process.pid}")
             return parse_one_collapsed(output, process_comm(ap_proc.process))
 
+    def _check_hotspot_error(self, ap_proc):
+        error_file = ap_proc.locate_hotspot_error_file()
+        if not error_file:
+            return
```

Comment:
For explicitness - can we log here? "No hs_err file found ..."

Reply:
I think that'd spam a lot. We'd want to log it only in case of non-zero exit status.

Reply:
Yes, I suppose we would. I suggest we do add it here because I prefer it to be spammy but "hermetic" in the sense that I know what happened. With the proc_connector PR, we will change it to test only upon a non-zero (or signal) exit code.

Reply:
For example, now I'm running it and the file isn't detected and I don't know why. An explicit message is useful

Reply:
OK